server/dummyData.js

```diff
@@ -26,315 +26,327 @@ module.exports = function() {
+              location1 = location;
             }
           });
   
-  let locationTwo = new Location();
-  let coordinateTwo = new Coordinate();
-  Location.findOne({'name': "Melbourne Airport"})
+  let location2 = new Location();
+  
+  var nameLocation2 = "Melbourne Airport";
+  Location.findOne({'name': nameLocation2})
           .populate('coordinates')
           .exec((error, location) => {
             if (!location) {
-              console.log('Adding Melbourne Airport...');
-              coordinateTwo.latitude = "-37.669012";
-              coordinateTwo.longitude = "144.841027";
-              coordinateTwo.save({}).catch();
-              locationTwo.name = "Melbourne Airport";
-              locationTwo.coordinates = coordinateTwo;
-              locationTwo.save({}).catch();
+              let coordinate2 = new Coordinate();
+               /* coordinate2 only used within location2 creation
+              no need to be set if location is already found
+              */
+              console.log('Adding', nameLocation2, '...');
+              coordinate2.latitude = "-37.669012";
+              coordinate2.longitude = "144.841027";
+              coordinate2.save({}).catch();
+              location2.name = nameLocation2;
+              location2.coordinates = coordinate2;
+              location2.save({}).catch();
             }
             else {
-              console.log('Melbourne Airport already exists');
-              locationTwo = location;
+              console.log("DB Error: Location \'", nameLocation2, "\' already exists.");
+              location2 = location;
             }
           });
 
-  let locationThree = new Location();
-  let coordinateThree = new Coordinate();
-  Location.findOne({'name': "Melbourne Airport"})
+  let location3 = new Location();
+
+  var nameLocation3 = "Perth Airport";
+  Location.findOne({'name': nameLocation3})
   .populate('coordinates')
   .exec((error, location) => {
     if (!location) {
-      console.log('Adding Perth Airport...');
-      coordinateThree.latitude = "-31.953512";
-      coordinateThree.longitude = "115.857048";
-      coordinateThree.save({}).catch();
-      locationThree.name = "Perth Airport";
-      locationThree.coordinates = coordinateThree;
-      locationThree.save({}).catch();
+      let coordinate3 = new Coordinate();
+      /* coordinate2 only used within location2 creation
+      no need to be set if location is already found
+      */
+     console.log('Adding', nameLocation3, '...');
+      coordinate3.latitude = "-31.953512";
+      coordinate3.longitude = "115.857048";
+      coordinate3.save({}).catch();
+      location3.name = nameLocation3;
+      location3.coordinates = coordinate3;
+      location3.save({}).catch();
     }
     else {
-      console.log('Perth Airport already exists');
-      locationThree = location;
+      console.log("DB Error: Location \'", nameLocation3, "\' already exists.");
```

Comment:
Can we remove the "DB Error"? Its not technically an error because we are trying to provide baseline data, and we are performing an explicit check to see if the data exists and acting accordingly